### DISCUSSION TITLE
Linux aarch64 builds

### DISCUSSION THREAD

**Discussion by aliosa27** [goastian/midori-desktop#54] [General]
Once upon a time there were aarch64 builds of this browser for linux, and the current documentation suggests that this is the case for Linux. There has not been an aarch64 build in quite a while, is it possible to get one?

**Comment by Ponchale** [comment]
We are working on the publication of a version for this architecture, we are waiting for devices to carry out the construction.

**Comment by Ponchale** [comment]
@aliosa27 Since version 11.2.1 it already supports the aarch64 architecture, we are already on 11.2.2, the tar is still available, and the .deb
